ipatests/test_integration/test_sssd.py

```diff
@@ -348,3 +350,179 @@ def get_cache_update_time(obj_kind, obj_name):
         finally:
             sssd_conf_backup.restore()
             tasks.clear_sssd_cache(self.master)
+
+
+@pytest.skipif("SSSD fix is not available on Fedora 27")
```

Comment:
The annotation should be `@pytest.mark.skip(reason=...")` or `@pytest.mark.skipif(condition, reason="...")`

Reply:
fixed it